style.css

```diff
@@ -657,3 +689,263 @@ textarea::placeholder {
+    flex: none;
+    margin-bottom: 24px;
+    width: 324px;
+    height: 248px;
+    padding-right: 20px;
+  }
+
+  .works-content h2 {
+    display: flex;
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-size: 32px;
+    font-weight: 400;
+    line-height: 138%;
+    margin-top: 0;
+    margin-bottom: 12px;
+  }
+
+  .works-content p {
+    display: flex;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 150%;
+    color: #091e42;
+    min-width: 400px;
+    width: 400px;
+  }
+
+  .works-content ul {
+    display: flex;
+    list-style-type: none;
+    justify-content: left;
+    width: 400px;
+  }
+
+  .center-content {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, auto);
+    grid-gap: 24px;
+    padding: 0 30px 40px 30px;
+  }
+
+  .about {
+    background-color: white;
+    width: 100%;
+  }
+
+  .about-header {
+    background-image: url("images/about_back_big.png");
+    background-position: right;
+    background-size: contain;
+  }
+
+  .about-header p {
+    margin-top: 0;
+    margin-bottom: 24px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 150%;
+    color: #f4f5f7;
+    width: 566px;
+  }
+
+  .major-menu {
+    display: flex;
+    flex-direction: row;
+    color: #fff;
+    padding: 0;
+    margin: 0;
+    align-items: flex-start;
+  }
+
+  .major-menu > li {
+    width: 33%;
+    padding-left: 5%;
+  }
+
+  .minor-menu {
+    align-items: flex-start;
+  }
+
+  .lang-first {
+    font-family: 'InaiMathi', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 138%;
+    color: #fff;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .lang-sec {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 138%;
+    color: #fff;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .contact-article {
+    display: flex;
+    flex-direction: row;
+    padding: 58px 80px 10px 20px;
+    gap: 24px;
+  }
+
+  .contact-article p {
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    font-family: 'inter', sans-serif;
+    line-height: 44px;
+    text-align: left;
+    color: #172b4d;
+    padding: 0 0 0 50px;
+    width: 455px;
+  }
+
+  .contact-form {
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 78px;
+    align-items: flex-start;
+    width: 100%;
+  }
+
+  .contact-form > input {
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 78px;
+    align-items: flex-start;
+    width: 100%;
+  }
+
+  .contact-form > textarea {
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 78px;
+    align-items: flex-start;
+    width: 100%;
+  }
+}
```

Comment:
- [ ] Your Mobile style has been affected with your desktop style. You're expected to ensure that the style for your desktop does not affected the style for the mobile. Kindly watch the screen recording below and check the Figma design for mobile and ensure your design remain as it was before you started the desktop version

### Your Mobile Design
[your-mobile.webm](https://user-images.githubusercontent.com/26597432/236643231-c1679b52-f101-41d9-be81-bed7583d1f44.webm)